tests/qxmppjingleiq/tst_qxmppjingleiq.cpp

```diff
@@ -59,72 +59,92 @@ void tst_QXmppJingleIq::testContent()
     const QByteArray xml(
         "<content creator=\"initiator\" name=\"voice\">"
         "<description xmlns=\"urn:xmpp:jingle:apps:rtp:1\" media=\"audio\">"
-        "<payload-type id=\"96\" name=\"speex\" clockrate=\"16000\"/>"
-        "<payload-type id=\"97\" name=\"speex\" clockrate=\"8000\"/>"
-        "<payload-type id=\"18\" name=\"G729\"/>"
-        "<payload-type id=\"0\" name=\"PCMU\"/>"
-        "<payload-type id=\"103\" name=\"L16\" channels=\"2\" clockrate=\"16000\"/>"
-        "<payload-type id=\"98\" name=\"x-ISAC\" clockrate=\"8000\"/>"
+        "<rtcp-mux/>"
+        "<payload-type id=\"96\"/>"
+        "<payload-type id=\"97\"/>"
         "</description>"
         "<transport xmlns=\"urn:xmpp:jingle:transports:ice-udp:1\""
         " ufrag=\"8hhy\""
         " pwd=\"asd88fgpdd777uzjYhagZg\">"
-        "<candidate component=\"1\""
-        " foundation=\"1\""
+        "<candidate component=\"0\""
         " generation=\"0\""
         " id=\"el0747fg11\""
-        " ip=\"10.0.1.1\""
-        " network=\"1\""
-        " port=\"8998\""
-        " priority=\"2130706431\""
-        " protocol=\"udp\""
+        " network=\"0\""
+        " port=\"0\""
+        " priority=\"0\""
         " type=\"host\"/>"
-        "<candidate component=\"1\""
-        " foundation=\"2\""
+        "<candidate component=\"0\""
         " generation=\"0\""
         " id=\"y3s2b30v3r\""
-        " ip=\"192.0.2.3\""
-        " network=\"1\""
-        " port=\"45664\""
-        " priority=\"1694498815\""
-        " protocol=\"udp\""
-        " type=\"srflx\"/>"
+        " network=\"0\""
+        " port=\"0\""
+        " priority=\"0\""
+        " type=\"host\"/>"
```

Comment:
why are you testing only with zeros (instead of realistic data)?

Reply:
This tests only the parts of the content because the candidate is already tested by `testCandidate()`. By doing that, we do not recursively duplicate tests and instead focus only on what has to be tested here.